Python/libraries/recognizers-number/recognizers_number/number/spanish/extractors.py

```diff
@@ -136,12 +136,12 @@ def _extract_type(self) -> str:
 
     def __init__(self):
         self.__regexes = [
-            ReVal(
-                re=SpanishNumeric.FractionNotationRegex,
-                val='FracNum'),
-            ReVal(
-                re=SpanishNumeric.FractionNotationWithSpacesRegex,
-                val='FracNum'),
+            # ReVal(
+            #     re=SpanishNumeric.FractionNotationRegex,
+            #     val='FracNum'),
+            # ReVal(
+            #     re=SpanishNumeric.FractionNotationWithSpacesRegex,
+            #     val='FracNum'),
             ReVal(
                 re=SpanishNumeric.FractionNounRegex,
                 val='FracSpa'),

```

Comment:
what is the rationale for keeping this regex?
If I'm not mistaken, this would transcript `dos tercios` as `1,666666`. I'm not sure this is what we want.
I've tried Android and `dos tercios` is left as text. We should check iOS as well.

Reply:
Yes, you right, we don't want this behaviour, I'm going to deactive all options relative to fractions.